config/vufind/config.ini

```diff
@@ -58,15 +58,16 @@ theme           = bootprint3
 ; actual theme name.  For example, http://library.myuniversity.edu/vufind?ui=theme1
 ; would load the myTheme1 theme with the setting shown below.  Note that the values
 ; of "standard" and "mobile" are reserved for the default and mobile themes defined
-; above.
-;alternate_themes = theme1:myTheme1,theme2:myTheme2
+; above. Example: theme1:myTheme1,theme2:myTheme2
+alternate_themes = custom:custom_tmpl
```

Comment:
My main concern here is that I don't want to surprise people with hidden features when they go to production. Some people might be annoyed if they suddenly learned that their production site could be forced into a weird-looking demo theme through the addition of an arbitrary GET parameter. It's no big deal, obviously, but I think it's a little more professional to make it opt-in. 

Another consideration: directly referencing the template theme in config.ini may encourage people to edit it directly, which as discussed elsewhere is really not what we want people to do.